### ISSUE TITLE
545408B8 - Grand Theft Auto: San Andreas

### ISSUE THREAD

**Issue by Blackbird88** [xenia-project/game-compatibility#186]
[Marketplace](http://marketplace.xbox.com/en-US/Product/Grand-Theft-Auto-San-Andreas/66acd000-77fe-1000-9115-d802545408b8)

Tested with benvanik/xenia@8046c19

* Crashes on startup
* Uses some weird XBLA file structure with multiple archives

[Log](https://gist.github.com/Blackbird88/e08d63c16714fb9fced5)

**Comment by TotalCaesar659** [comment]
Tested on (https://github.com/benvanik/xenia/commit/abf162c0e32889b37461374b1d25ef9b5bba2648)

# Issues:
There is white screen only.

# Log:
[Log file](https://drive.google.com/file/d/0B7KpdfS_wYnZdDVCY0k0WThwSms/view)

# Screenshot(s):
![0000](https://cloud.githubusercontent.com/assets/14265316/17650557/6bfee976-6259-11e6-89db-cae965186bd1.jpg)

# Labels:
(state-nothing)

**Comment by Blackbird88** [comment]
Yeah so it's still same and?

**Comment by TotalCaesar659** [comment]
Yeah, it's same. I just added screenshot for visibility and updated log file.

**Comment by spectrumm** [comment]
[New log](https://gist.github.com/spectrumm/1e0347fcbbdaf8f6c6b1a6aea5ed1b3b)

**Comment by Parovozik** [comment]
Tested on (https://github.com/benvanik/xenia/commit/e8f55d8ff276cce22cf7c44fa95c2fac974ddbef)

# Issues:
Broken all models of characters (view as triangle or just stripes)
Sound - OK
Save-states - OK
Performance - OK
Stable ingame

# LOGs:
[GTA SA HD1 Vulkan.txt](https://github.com/xenia-project/game-compatibility/files/1026038/GTA.SA.HD1.txt)
[GTA SA HD2 Vulkan.txt](https://github.com/xenia-project/game-compatibility/files/1026040/GTA.SA.HD2.txt)

# Video:
[YouTube_GTA_SanAndreas_Vulkan](https://www.youtube.com/watch?v=XZNKaCbVtBg)

# Screnshots:
![xenia gta sa vk mp4_snapshot_00 49_ 2017 05 24_18 07 24](https://cloud.githubusercontent.com/assets/19775644/26410736/cd43713e-40ac-11e7-8ad8-b31ad2cac8e7.jpg)
![xenia gta sa vk mp4_snapshot_01 29_ 2017 05 24_18 07 48](https://cloud.githubusercontent.com/assets/19775644/26410739/cf41a640-40ac-11e7-827a-faff7c1e06eb.jpg)
![xenia gta sa vk mp4_snapshot_12 06_ 2017 05 24_18 08 13](https://cloud.githubusercontent.com/assets/19775644/26410746/d25fffde-40ac-11e7-9881-d30078ff368a.jpg)
![xenia gta sa vk mp4_snapshot_15 15_ 2017 05 24_18 08 01](https://cloud.githubusercontent.com/assets/19775644/26410750/d46e52bc-40ac-11e7-943d-8daf8a40f75f.jpg)

# Specs:
CPU Intel i5-3570K @ 3.6 GHz
GPU GTX 1050ti 4GB
RAM 8GB DDR3 (DC)
Windows 10 x64

# Labels:
gpu-corrupt-drawing
state-gameplay

**Comment by Parovozik** [comment]
Characters almost perfect, but still some textures without transparency
Sound - OK
Performance - perfect
![xenia gta sa vk mp4_snapshot_07 07_ 2018 02 23_20 15 27](https://user-images.githubusercontent.com/19775644/36607034-792fc770-18d6-11e8-8732-63fe82895806.jpg)
![xenia gta sa vk mp4_snapshot_13 42_ 2018 02 23_20 15 13](https://user-images.githubusercontent.com/19775644/36607037-7fa89988-18d6-11e8-849d-78906d723442.jpg)
![xenia gta sa vk mp4_snapshot_13 40_ 2018 02 23_20 15 00](https://user-images.githubusercontent.com/19775644/36607043-826378e6-18d6-11e8-93dc-9f692d6ba3cd.jpg)

**Comment by boiiiiboiiii** [comment]
Any updates on the graphical state of the game?

**Comment by uvraj** [comment]
Characters ALMOST perfect (Sometimes flash). Transparency issues and image sometimes becomes white in cutscenes.+UI bugs.  And image flips sometimes....... Map colours are messed up.
![ui issue](https://user-images.githubusercontent.com/30289260/39967429-6ebc8710-56bb-11e8-88c7-29e5bee115df.PNG)
![ui issues](https://user-images.githubusercontent.com/30289260/39967430-6eda0da8-56bb-11e8-8c1d-066d83137d3a.PNG)
![map colours messed up](https://user-images.githubusercontent.com/30289260/39967437-8448f244-56bb-11e8-983d-1c8263d82705.PNG)

**Comment by BiwwytheBoi** [comment]
I got videos of the game running on an actual build and a custom build on D3D12
Specs:
GTX 670 2gb
AMD A10-7800
8GB DDR3
Windows 10 x64 

Any stutter seen is either the recording or the game

Actual: 
https://www.youtube.com/watch?v=8m07cgEBSyY
Custom:
https://www.youtube.com/watch?v=Qk3bEAhxZ-s

**Comment by BiwwytheBoi** [comment]
Should I try completing the game?

**Comment by GrandGTA** [comment]
https://www.youtube.com/watch?v=jhVdb4K1M-M&t&ab_channel=VadimM Its impossible to complete this game because the game itself has bugs... Not just the emulator that has bugs, but the game also has bugs.... Someone might get confused saying "the character's face has bugs" and the Xenia devs might think that its a bug from the emulator when its just a bug from the game itself...

**Comment by Mew99-gtich** [comment]
L

**Comment by Margen67** [comment]
Xenia Canary issue: https://github.com/xenia-canary/game-compatibility/issues/15
